crates/bevy_core_pipeline/src/core_3d/camera_3d.rs

```diff
@@ -5,20 +5,46 @@ use bevy_render::{
     camera::{Camera, CameraRenderGraph, Projection},
     extract_component::ExtractComponent,
     primitives::Frustum,
-    render_resource::LoadOp,
+    render_resource::{LoadOp, TextureUsages},
     view::VisibleEntities,
 };
 use bevy_transform::prelude::{GlobalTransform, Transform};
 use serde::{Deserialize, Serialize};
 
 /// Configuration for the "main 3d render graph".
-#[derive(Component, Reflect, Clone, Default)]
+#[derive(Component, Reflect, Clone)]
 #[reflect(Component)]
 pub struct Camera3d {
     /// The clear color operation to perform for the main 3d pass.
     pub clear_color: ClearColorConfig,
     /// The depth clear operation to perform for the main 3d pass.
     pub depth_load_op: Camera3dDepthLoadOp,
+    /// The texture usages for the depth texture created for the main 3d pass.
+    pub depth_texture_usages: Camera3dDepthTextureUsage,
+}
+
+impl Default for Camera3d {
+    fn default() -> Self {
+        Self {
+            clear_color: ClearColorConfig::Default,
+            depth_load_op: Default::default(),
+            depth_texture_usages: TextureUsages::RENDER_ATTACHMENT.into(),
+        }
+    }
+}
+
+#[derive(Clone, Copy, Reflect)]
+pub struct Camera3dDepthTextureUsage(u32);
```

Comment:
Creating a custom class to wrap the `TextureUsage` so that it implements `Reflect`